ml-agents/mlagents/trainers/tests/mock_brain.py

```diff
@@ -31,18 +32,17 @@ def create_mock_brainparams(
     mock_brain.return_value.vector_observation_space_size = (
         vector_observation_space_size
     )
-    camrez = CameraResolution(height=84, width=84, num_channels=3)
+    camrez = Tensor3DShape(height=84, width=84, num_channels=3)
     mock_brain.return_value.camera_resolutions = [camrez] * number_visual_observations
     mock_brain.return_value.vector_action_space_size = vector_action_space_size
     mock_brain.return_value.brain_name = "MockBrain"
     return mock_brain()
```

Comment:
What is mock brain mocking ? Would it make more sense to have this return a real BehaviorSpec object ?

Reply:
I swear I got this one, good catch - it is gone now. Was no longer used